npachpande/async/miawCustomPrechat/customPrechatForm.html

```diff
@@ -0,0 +1,12 @@
+<template>
+    <template class="slds-m-around_medium" for:each={fields} for:item="field">
+        <c-custom-prechat-form-field key={field.name} 
```

Comment:
created a separate component to render each formField, mainly to demonstrate how a customer can distinguish between a text field vs a dropdown field.